### PR TITLE
Messenger track kit

### PR DESCRIPTION
**Pull request by liach**
Supersedes #926. Done.

### CODE REVIEW

File: src/main/java/mods/railcraft/common/blocks/tracks/outfitted/kits/TrackKitMessenger.java [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+                text = new TextComponentString(heldItem.getDisplayName());
+            }
+        }
+        return super.blockActivated(player, hand, heldItem);
+    }
+
+    @Override
+    public void onMinecartPass(EntityMinecart cart) {
+        long time = cart.worldObj.getWorldTime();
+        if (lastCart != null && lastCart.get() == cart) {
```
Comment (liach) [inline src/main/java/mods/railcraft/common/blocks/tracks/outfitted/kits/TrackKitMessenger.java]: Probably will add a train mode that notifies all the riders of the train at the same time. How do you think?

**Comment by Ristellise** [comment]
it would be best if it messaged the player instead of doing it publicly or message it in a radius...

**Comment by liach** [comment]
@Joshwoo70 It is already messaging each player on the cart instead of in a radius.

**Comment by liach** [comment]
@CovertJaguar Done.

**Comment by Ristellise** [comment]
:shipit: :confetti_ball:

**Comment by Ristellise** [comment]
aww the first message should say that jermt fuel can't melt dank memes XD anyways.. i find the texture a bit too vibrant.. and doesn't fit the MC feel.

**Comment by CovertJaguar** [comment]
I'll admit, I don't really get the texture.

**Comment by liach** [comment]
It is a speaker

**Comment by liach** [comment]
The new texture is adjusted to be more like a speaker.

**Comment by Ristellise** [comment]
the speaker should bs facing upwards XD

**Comment by liach** [comment]
@CovertJaguar Build failed because the `mc-1.10.2` branch did not build.

**Comment by CovertJaguar** [comment]
@liach is this ready for merging?

**Comment by liach** [comment]
Yes. No longer WIP~~

**Review by CovertJaguar — APPROVED** [review]
👍

**Pull request merged**